sdk/lib/drivers/rdbsslib/rdbss.c

```diff
@@ -26,6 +26,12 @@
 
 /* INCLUDES *****************************************************************/
 
+/* Starting with NT6, RDBSS got totally rewritten to change the way it works */
+#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
+#undef _WIN32_WINNT
+#define _WIN32_WINNT _WIN32_WINNT_WS03
```

Comment:
Once again, WIN32 is no point in the kernel.

Reply:
The correct check is checking `NTDDI_VERSION` against `NTDDI_WS03` or `NTDDI_LONGHORN` (or `NTDDI_VISTA` for that matter).